tv_apps/smart-home/bower_components/smart-button/style.css

```diff
@@ -492,6 +492,15 @@ smart-button[type="rounded-corner-text"].primary.pressed {
   background-color: #0097b5;
 }
 
+smart-button[type="rounded-corner-text"].danger,
+smart-button[type="rounded-corner-text"].danger.focused {
+  background-color: #e63d2f;
+}
+
+smart-button[type="rounded-corner-text"].danger.pressed {
+  background-color: #ac2e23;
+}
```

Comment:
Please update `smart-button` repository if you want to make change of this.

Or you can just add them in card_picker.css.